examples/gpt_actions_library/gpt_action_bigquery.ipynb

```diff
@@ -0,0 +1,276 @@
+    "            type: string\n",
+    "      requestBody:\n",
+    "        required: true\n",
+    "        content:\n",
+    "          application/json:\n",
+    "            schema:\n",
+    "              type: object\n",
+    "              properties:\n",
+    "                query:\n",
+    "                  type: string\n",
+    "                  description: The SQL query string.\n",
+    "                useLegacySql:\n",
+    "                  type: boolean\n",
+    "                  description: Whether to use legacy SQL.\n",
+    "                  default: false\n",
+    "      responses:\n",
+    "        '200':\n",
+    "          description: Successful query execution.\n",
+    "          content:\n",
+    "            application/json:\n",
+    "              schema:\n",
+    "                type: object\n",
+    "                properties:\n",
+    "                  kind:\n",
+    "                    type: string\n",
+    "                    example: \"bigquery#queryResponse\"\n",
+    "                  schema:\n",
+    "                    type: object\n",
+    "                    description: The schema of the results.\n",
+    "                  jobReference:\n",
+    "                    type: object\n",
+    "                    properties:\n",
+    "                      projectId:\n",
+    "                        type: string\n",
+    "                      jobId:\n",
+    "                        type: string\n",
+    "                  rows:\n",
+    "                    type: array\n",
+    "                    items:\n",
+    "                      type: object\n",
+    "                      properties:\n",
+    "                        f:\n",
+    "                          type: array\n",
+    "                          items:\n",
+    "                            type: object\n",
+    "                            properties:\n",
+    "                              v:\n",
+    "                                type: string\n",
+    "                  totalRows:\n",
+    "                    type: string\n",
+    "                    description: Total number of rows in the query result.\n",
+    "                  pageToken:\n",
+    "                    type: string\n",
+    "                    description: Token for pagination of query results.\n",
+    "        '400':\n",
+    "          description: Bad request. The request was invalid.\n",
+    "        '401':\n",
+    "          description: Unauthorized. Authentication is required.\n",
+    "        '403':\n",
+    "          description: Forbidden. The request is not allowed.\n",
+    "        '404':\n",
+    "          description: Not found. The specified resource was not found.\n",
+    "        '500':\n",
+    "          description: Internal server error. An error occurred while processing the request."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Authentication Instructions"
```

Comment:
ditto on the structure / friendliness of this doc as a standalone thing. A single line here (or at the very top being like "this does not require authentication" would better prepare the reader to understand why this is all "None".

Reply:
100% agree -- updated that in the latest commit (and in the weather file, I just removed the irrelevant content + added that single note at the top)